webpack.config.js

```diff
@@ -54,7 +54,7 @@ const config = {
         new MiniCssExtractPlugin({
             filename: 'style.[contenthash].css',
         }),
-        new webpack.EnvironmentPlugin({ ...process.env }),
+        new Dotenv(),
```

Comment:
@joaopslins eu li aqui no readme deles, que pra pegar as do sistema tem que habilitar essa opçāo: `systemvars (false) - Set to true if you would rather load all system variables as well (useful for CI purposes).`
Como parametro dessa conf

Reply:
Hummm verdade, tava esperando abrir o PR pra testar isso. [Inclusive o preview falhou](https://deploy-preview-111--issueai.netlify.com)

Vou modificar